src/Groups/gsets.jl

```diff
@@ -709,9 +710,10 @@ is_conjugate(Omega::GSet, omega1, omega2) = omega2 in orbit(Omega, omega1)
     is_conjugate_with_data(Omega::GSet, omega1, omega2)
 
 Determine whether `omega1`, `omega2` are in the same orbit of `Omega`.
-If yes, return `true, g` where `g` is an element in the group `G` of
+If yes, return `(true, g)` where `g` is an element in the group `G` of
 `Omega` that maps `omega1` to `omega2`.
-If not, return `false, nothing`.
+If not, return `(false, nothing)`.
```

Comment:
This function apparently does not return `nothing` (in the Julia sense), but a group element.

Aside: I'm wondering whether the other two should do the same in terms of type stability?

Reply:
I would say yes, maybe just `one(G)`

Reply:
I don't like this, I think it should return `nothing`; I guess it's returning `false` anyway, but returning the identity element to me should only happen if `omega1 == omega2`.

Reply:
Well, it's common practice in OSCAR, to return a second "dummy" element for type stability in case the first argument is `false`. But no strong argument from my side.